### PR TITLE
[batch] Progress bar for jobs

### PR DESCRIPTION
**Pull request by CDiaz96**

### CODE REVIEW

**Comment by jigold** [comment]
Can you fix the following pylint formatting errors?

```
batch/front_end/front_end.py:1455:1: W293 blank line contains whitespace
batch/front_end/front_end.py:1462:1: W293 blank line contains whitespace
batch/front_end/front_end.py:1463:13: W291 trailing whitespace
```

**Comment by CDiaz96** [comment]
@jigold I went ahead and fixed the formmating issues. Let me know if anything else pops up but I think it should be good now.

File: web_common/web_common/styles/main.scss [hunk 1]
```diff
@@ -452,3 +452,39 @@ td a.fill-td {
...
+div .bar {
+  display: inline-block;
+  margin: 0;
+}
+
+.textbox {
+  display: none;
+}
+
+.progress_bar {
```
Comment (jigold) [inline web_common/web_common/styles/main.scss:466]: Do we need to make these changes specific to the progress bar? Something like `.progress_bar div .bar`?

**Review by jigold — CHANGES_REQUESTED** [review]

File: web_common/web_common/styles/main.scss [hunk 2]
```diff
@@ -452,3 +452,9 @@ td a.fill-td {
   width: 100%;
   height: 100%;
 }
+//Progress bar css
+
+div .bar {
```
Comment (jigold) [inline web_common/web_common/styles/main.scss:457]: I'm sorry. I should have been more clear. Are you sure we need this change? If so, can we add something that corresponds to the id of the div the plot is in (`plotly-timeseries`)? I think it's this:

```css
#plotly-timeseries .bar {
    display: inline-block;
    margin: 0;
}
```

**Review by jigold — CHANGES_REQUESTED** [review]

File: web_common/web_common/styles/main.scss (hunk 2 above)
Comment (CDiaz96) [reply web_common/web_common/styles/main.scss:457]: Oh Okay. I get what you mean. I am going to try it but I think that specific id is directly correlated with Plotly and the css it has attached to it, so we might not need any of the CSS specifically. But let me run it and make sure.

Comment (jigold) [reply web_common/web_common/styles/main.scss:457]: That's what I thought. I don't think you need this at all.

**Review by jigold — APPROVED** [review]

**Pull request merged**